engine/src/main/java/org/terasology/rendering/ShaderManagerLwjgl.java

```diff
@@ -105,31 +92,28 @@ public ShaderManagerLwjgl() {
-        defaultTexturedShaderProgram = prepareAndStoreShaderProgramInstance("defaultTextured", new ShaderParametersDefault());
+        defaultShaderProgram = prepareAndStoreShaderProgramInstance("default", new ShaderParametersEmpty());
+        defaultTexturedShaderProgram = prepareAndStoreShaderProgramInstance("defaultTextured", new ShaderParametersEmpty());
 
         // TODO: Find a better way to do this
         prepareAndStoreShaderProgramInstance("post", new ShaderParametersPost());
         prepareAndStoreShaderProgramInstance("ssao", new ShaderParametersSSAO());
-        prepareAndStoreShaderProgramInstance("lightShafts", new ShaderParametersLightShafts());
-        prepareAndStoreShaderProgramInstance("sobel", new ShaderParametersSobel());
-        prepareAndStoreShaderProgramInstance("initialPost", new ShaderParametersInitialPost());
-        prepareAndStoreShaderProgramInstance("prePostComposite", new ShaderParametersPrePostComposite());
-        prepareAndStoreShaderProgramInstance("highPass", new ShaderParametersDefault());
-        prepareAndStoreShaderProgramInstance("blur", new ShaderParametersDefault());
-        prepareAndStoreShaderProgramInstance("downSampler", new ShaderParametersDefault());
-        prepareAndStoreShaderProgramInstance("toneMapping", new ShaderParametersToneMapping());
-        prepareAndStoreShaderProgramInstance("sky", new ShaderParametersSky());
-        prepareAndStoreShaderProgramInstance("chunk", new ShaderParametersChunk());
-        prepareAndStoreShaderProgramInstance("particle", new ShaderParametersParticle());
-        prepareAndStoreShaderProgramInstance("block", new ShaderParametersBlock());
-        prepareAndStoreShaderProgramInstance("shadowMap", new ShaderParametersShadowMap());
+        prepareAndStoreShaderProgramInstance("lightShafts", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("sobel", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("initialPost", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("prePostComposite", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("highPass", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("blur", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("downSampler", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("toneMapping", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("sky", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("chunk", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("particle", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("shadowMap", new ShaderParametersEmpty());
         prepareAndStoreShaderProgramInstance("debug", new ShaderParametersDebug());
```

Comment:
Nope, just boring. The remaining ShaderParameters are used in multiple Nodes, and I wanted feedback before going ahead and copying all the code from one class to multiple Nodes.